configuration/configure-unhealthy-cluster-deletion.adoc

```diff
@@ -0,0 +1,21 @@
+:_content-type: ASSEMBLY
+[id="configure-unhealthy-cluster-deletion"]
+= Configuring automatic deletion of unhealthy clusters
+include::modules/common-attributes.adoc[]
+:context: configure-unhealthy-cluster-deletion
+
+toc::[]
+
+[role="_abstract"]
+{product-title} ({product-title-short}) provides the ability to automatically delete unhealthy clusters from your system. If this feature is enabled, when Central has been unable to reach Sensor in a cluster for a configured period of time, the cluster is considered unhealthy (decommissioned). Unhealthy clusters are then automatically deleted based on your configuration settings. With automatic deletion of unhealthy clusters, you can easily monitor active clusters only. You can configure cluster removal settings by using the {product-title-short} portal or the application programming interface (API). You can specify not to delete an unhealthy cluster by adding a label for the cluster when configuring this feature.
```

Comment:
> Avoid using parentheses in general text because text in parentheses is decreased in importance.

I think "you can monitor only active clusters" might be a better choice.

> Perceived benefits or characteristics of a product: Avoid phrases such as "simple to use," "effortless," "user-friendly," "future-proof," and "easy to understand." Users might not find these statements true, and inflated language can reduce the credibility of the information.